package.json

```diff
@@ -2,7 +2,7 @@
   "name": "pep",
   "title": "jquery.pep.js",
   "description": "Kinetic drag for mobile & desktop",
-  "version": "0.6.8",
+  "version": "0.6.9",
```

Comment:
Please revert this line since I cannot publish a new NPM version after merging this PR.